src/pages/resources/migration/android/migrate-to-2x.md

```diff
@@ -0,0 +1,272 @@
+import Tabs from './tabs/migrate-to-2x.md'
+
+# Migrate to Adobe Experience Platform Mobile 2.x SDKs for Android
```

Comment:
For consistency with the index page: `Migrate to Adobe Experience Platform 2.x SDKs for Android`